webapp_v2/src/components/singlestudy/NavHeader.tsx

```diff
@@ -0,0 +1,290 @@
+                  <ListItemIcon>
+                    <ArchiveOutlinedIcon sx={{ color: 'action.active', width: '24px', height: '24px' }} />
+                  </ListItemIcon>
+                  <ListItemText>
+                    {t('studymanager:archive')}
+                  </ListItemText>
+                </MenuItem>
+                )}
+              </div>
+            )}
+            {
+              study?.managed && (
+              <MenuItem onClick={() => { deleteStudy(study); handleClose(); }}>
+                <ListItemIcon>
+                  <DeleteOutlinedIcon sx={{ color: 'error.light', width: '24px', height: '24px' }} />
+                </ListItemIcon>
+                <ListItemText sx={{ color: 'error.light' }}>
+                  {t('studymanager:delete')}
+                </ListItemText>
+              </MenuItem>
+              )}
+          </Menu>
+        </Box>
+      </Box>
+      {study && (
+      <Box my={2} width="100%" display="flex" flexDirection="row" justifyContent="flex-start" alignItems="center" boxSizing="border-box">
+        <Box display="flex" flexDirection="row" justifyContent="flex-start" alignItems="center">
+          <ScheduleOutlinedIcon sx={{ color: 'text.secondary', mr: 1 }} />
+          <TinyText>
+            {convertUTCToLocalTime(study.creationDate)}
+          </TinyText>
+        </Box>
+        <Box mx={3} display="flex" flexDirection="row" justifyContent="flex-start" alignItems="center">
+          <UpdateOutlinedIcon sx={{ color: 'text.secondary', mr: 1 }} />
+          <TinyText>
+            {buildModificationDate(study.modificationDate, t)}
+          </TinyText>
+        </Box>
+        <StyledDivider />
+        {parent && (
+        <Box mx={3} display="flex" flexDirection="row" justifyContent="flex-start" alignItems="center">
+          <AltRouteOutlinedIcon sx={{ color: 'text.secondary', mr: 1 }} />
+          <LinkText to={`/studies/${parent.id}`}>{t('singlestudy:parentStudy')}</LinkText>
+        </Box>
+        )}
+        {childrenTree && (
+        <Box mx={3} display="flex" flexDirection="row" justifyContent="flex-start" alignItems="center">
+          <AccountTreeOutlinedIcon sx={{ color: 'text.secondary', mr: 1 }} />
+          <TinyText>
+            {childrenTree.children.length}
+          </TinyText>
+        </Box>
+        )}
+        <StyledDivider />
+        <Box mx={3} display="flex" flexDirection="row" justifyContent="flex-start" alignItems="center">
+          <PersonOutlineOutlinedIcon sx={{ color: 'text.secondary', mr: 1 }} />
+          <TinyText>
+            {study?.owner.name}
+          </TinyText>
+        </Box>
+        <Box mx={3} display="flex" flexDirection="row" justifyContent="flex-start" alignItems="center">
+          <SecurityOutlinedIcon sx={{ color: 'text.secondary', mr: 1 }} />
+          <TinyText>
+            {study?.publicMode === 'NONE' ? t('singlestudy:publicModeAnybody') : t('singlestudy:publicModeNone')}
```

Comment:
use the existing trad for public modes : 
"singlestudy:nonePublicModeText", "singlestudy:readPublicModeText", "singlestudy:executePublicModeText", "singlestudy:editPublicModeText",  "singlestudy:fullPublicModeText",